examples/opencv/README.md

```diff
@@ -0,0 +1,136 @@
+# Distributed Image Processing with OpenCV and Pachyderm
+
+## Intro
+
+In this tutorial we're going to do edge detection using the Canny edge
+detection algorithm implemented in OpenCV. We'll be deploying our code as a
+Pachyderm pipeline which means it will be both streaming and distributed.
+
+This tutorial assumes that you've already got a Pachyderm cluster up and
+running and that you can talk to it with pachctl. You'll know it's working if
+`pachctl version` returns without any errors. If not head on over to the [setup
+guide](/doc/deploying_setup.md) to get a cluster up and running.
+
+## Load Images Into Pachyderm
+The first thing we'll need to do is create a repo to store our images in, we'll
+call the repo "images".
+
+```sh
+$ pachctl create-repo images
+```
+
+Now we need to put some images in that repo. You can do this with `put-file`:
+
+```sh
+$ pachctl put-file images master -c -i examples/opencv/images.txt
+```
+
+With this command you're telling Pachyderm that you want to put files in the
+`images` repo on the branch `master`. You're also passing two flags, `-c` means
+that you want Pachyderm to start and finish the commit for you. If you wanted
+to have multiple `put-files` write to the same commit you'd use `start-commit`
+and `finish-commit`. `-i` is where you actually specify the images you're
```

Comment:
`-i` lets you specify a line-delimited input file. Each line can be a URL to scrape or a local file to add. In our case, we've got a few image URLs that we'll add to Pachyderm.